package.json

```diff
@@ -24,7 +24,7 @@
     },
     "scripts": {
         "start": "react-scripts start",
-        "build": "react-scripts build",
+        "build": "CI=false react-scripts build",
```

Comment:
just curious, what happens otherwise?

Reply:
Warnings get treated as errors if CI is set to true (which is the default behavior) causing our builds to fail. As and when we iterate over our UI, these warnings should go away 😄

Reply:
maketh sense